.github/workflows/ci-components.yml

```diff
@@ -46,14 +46,14 @@ jobs:
       fail-fast: false
       matrix:
         try-scenario:
-         # - ember-lts-3.24
           - ember-lts-3.28
+          - ember-lts-4.4
+          - ember-lts-4.8
           - ember-release
           - ember-beta
-         # - ember-canary
-         # - ember-classic
-         # - embroider-safe
-         # - embroider-optimized
+          - ember-canary
```

Comment:
We had discussed excluding this some time ago as any failures here would likely not be very actionable for us, especially given how often our test suite runs.

Reply:
ok. What do you think about having a regular schedule to run against canary, just so we can get a heads up on any issues that might be headed our way? Canary is supposed to be an early warning signal (either to fix something in your app, or report the issue so it can be addressed in Ember proper). WDYT?

Reply:
🤷 we've been running the beta tests this entire time and I don't think we've seen a single instance of any issues being raised in that scenario, so canary seems unlikely to provide actionable feedback for us.